analyses/molecular-subtyping-pathology/README.md

```diff
@@ -42,4 +45,4 @@ bash run-subtyping-aggregation.sh
 
 `02-incorporate-clinical-feedback.Rmd` incorporate clincally reviewed subtypes for PNOC003 samples and update to the clinically reviewed subtype if they are different from the subtype from `molecular-subtyping-HGG`
 
-`03-incorporate-pathology-feedback.Rmd` incorporates pathology feedback for specific samples when the labels for those samples either need to be updated as a result of molecular subtyping or deviate from the logic in upstream molecular subtyping modules. The output is an updated version of the table from `01-compile-subtyping-results.Rmd` (`results/compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv`).
+`03-incorporate-pathology-feedback.Rmd` incorporates pathology feedback for specific samples when the labels for those samples either need to be updated as a result of molecular subtyping, molecular abberations were not sufficient or deviate from the logic in upstream molecular subtyping modules. The output is an updated version of the table from `01-compile-subtyping-results.Rmd` (`results/compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv`).
```

Comment:
```suggestion
`03-incorporate-pathology-feedback.Rmd` incorporates pathology feedback for specific samples when the labels for those samples either need to be updated as a result of molecular subtyping, because the molecular aberrations were not sufficient, or because the subtypes deviate from the logic in upstream molecular subtyping modules.
 The output is an updated version of the table from `01-compile-subtyping-results.Rmd` (`results/compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv`).
```

This sentence is pretty long that I'm not sure exactly what the meaning is. Can you break it up and describe the scenarios in which samples get updated in this notebook?